dotnet/src/dotnetframework/GxClasses/Middleware/GXHttpServices.cs

```diff
@@ -582,6 +582,12 @@ protected override GAMSecurityLevel IntegratedSecurityLevel
 	internal class GXOAuthAccessToken : GXHttpHandler, IRequiresSessionState
 	{
 		static readonly ILog log = log4net.LogManager.GetLogger(typeof(GeneXus.Http.GXOAuthAccessToken));
+		const string OTP_USER_ACCESS_CODE_SENT = "400";
+		const string TFA_USER_MUST_VALIDATE = "410";
```

Comment:
The [spec](https://iwiki.genexus.com/wiki.aspx?27384) seems to say that the code should be "420". Make sure you are using the correct code.

Reply:
Spec has been fixed.